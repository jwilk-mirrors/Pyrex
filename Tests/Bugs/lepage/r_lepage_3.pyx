cdef class Grail:

	def __call__(self, x, y, z):
		print "Grail called with:", x, y, z
