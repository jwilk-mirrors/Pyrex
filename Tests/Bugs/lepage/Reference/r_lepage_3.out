Grail called with: spam 42 ['tomato', 'sandwich']
